packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -25,6 +25,14 @@ export const prodParasKusama: EndpointOption[] = [
       OnFinality: 'wss://altair.api.onfinality.io/public-ws'
     }
   },
+  {
+    info: 'amplitude',
+    homepage: 'https://pendulumchain.org/',
```

Comment:
@gonzamontiel Shouldn't this be 'https://pendulumchain.org/amplitude'?

Reply:
```suggestion
    homepage: 'https://pendulumchain.org/amplitude',
```

Reply:
thanks, fixed : )